changelogs/unreleased/gh-8967-creds-postope-sync-when-RO-till-RW.md

```diff
@@ -0,0 +1,5 @@
+## feature/config
+
+* If Tarantool is in Read Only mode, credentials are now synchronised
+  in the background when switched to Read Write mode, instead of being
+  skipped (gh-8967).
```

Comment:
```suggestion
* On read-only instances, Tarantool now synchronizes credentials
  in the background when switching to read-write mode instead of
  skipping the synchronization (gh-8967).
```

Reply:
Applied.